<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputPieceDialog</class>
 <widget class="QDialog" name="InputPieceDialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>93</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Choose piece</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QPushButton" name="knightButton">
     <property name="minimumSize">
      <size>
       <width>75</width>
       <height>75</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>36</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">qproperty-icon: url(:/Images/WhitePieces/Rook.png)</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>70</width>
       <height>70</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="bishopButton">
     <property name="minimumSize">
      <size>
       <width>75</width>
       <height>75</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>36</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">qproperty-icon: url(:/Images/WhitePieces/Bishop.png)</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>70</width>
       <height>70</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="rookButton">
     <property name="minimumSize">
      <size>
       <width>75</width>
       <height>75</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>36</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">qproperty-icon: url(:/Images/WhitePieces/Rook.png)</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>70</width>
       <height>70</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="queenButton">
     <property name="minimumSize">
      <size>
       <width>75</width>
       <height>75</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>36</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">qproperty-icon: url(:/Images/WhitePieces/Queen.png)</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>70</width>
       <height>70</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
